src/config-loader/config-loader.coffee

```diff
@@ -93,6 +94,7 @@ exports.setup = (app) ->
 				Promise.map data.models, (model) ->
 					if model.modelText?
 						apiRoute = '/' + model.apiRoot + '/*'
+						app.post('/' + model.apiRoot + '/([\$])canAccess', sbvrUtils.authorizationMiddleware, resourceaccess.canAccessV1Request)
```

Comment:
This should match what? `/canAccess` and `/$canAccess`?

Reply:
This matches only `/$canAccess`.

Reply:
Cool, why not then:

```
'/' + model.apiRoot + '/$canAccess'
```

Reply:
I had to look this up, but if you want to use `$` in a path in an express route you need to escape it like this. 

"If you need to use the dollar character ($) in a path string, enclose it escaped within ([ and ]). For example, the path string for requests at “/data/$book”, would be “/data/([\$])book”." (see: https://expressjs.com/en/guide/routing.html)